### ISSUE TITLE
Report an error when an imported field value does not exist

### ISSUE THREAD

**Issue by richardbank** [Signbank/NGT-signbank#96]
When using the Import CSV script, it is possible to import field values that do not exist. Currently, Signbank behaves somewhat erratic when encountering such cases. See the following examples (in all cases, the string to change to includes a trailing space, see #95):
* Strong Hand changes from #B (90) to #B (0).
* Strong Hand changes from L (9) to L (0).
* Strong Hand changes from Beak-spread-open (87) to Beak-spread-open (4).
* Strong Hand changes from #I (98) to #I (6).

In the first two lines, a value is assigned to machine value '0'. In the third line, a value is assigned to machine value '4', which is already taken by another value, 'Money'. Similarly, in the fourth line, machine value '6' is already taken by the value 'B'.

Of course, removing trailing spaces solves this issue, but there should be an error handler to catch these instances. (and you may want to know why Signbank behaves this way :-)

**Comment by Woseseltops** [comment]
Fixed: the code that compares the csv to the existing database now raises an exception if it cannot find a machine value that matches a corresponding human value. If an error is reaised, the template shows an error instead of detected changes.

I did not succeed in discovering where the unexpected machine values come from. @richardbank , can you help me reproduce the behavior (if it still occurs after this fix)?

**Comment by richardbank** [comment]
I imported a new set of changes; some errors occurred, but all were explainable and were our own fault. Signbank behaved as expected. I will close this issue.

**Comment by richardbank** [comment]
I get a new kind of error when a to-be-imported handshape code does not exist as a field choice. There used to be a neat display of the found inconsistencies, now I get the standard Django error again:

##### DoesNotExist at /signs/import_csv/
###### Gloss matching query does not exist.

Request Method: 	POST
Request URL: 	http://signbank.science.ru.nl/signs/import_csv/
Exception Type: 	DoesNotExist
Exception Value: 	Gloss matching query does not exist.
Exception Location: 	/var/www2/signbank/live/virtualenv/signbank/lib/python2.7/site-packages/django/db/models/query.py in get, line 307

etc.

Please fix.

**Comment by Woseseltops** [comment]
Cool feature we describe in this issue, I completely forgot about it :). Anyway, I made my own testcase, but for me the error message still works as described:

> At wesseltest (3107), could not find option Wesselisdebeste for Named Entity

Can I have your csv, so I can reproduce the problem?

**Comment by richardbank** [comment]
Here's one. It's different from what I used earlier, but raises the same error.

[dictionary-export2.zip](https://github.com/Signbank/NGT-signbank/files/210054/dictionary-export2.zip)

**Comment by Woseseltops** [comment]
Thanks, I got the error as well! This file got a huge number of problems of two categories: all field values have dashes where underscores are expected (B-bent instead of B_bent), and it has references to glosses that are deleted. The first problem made the second problem less visible, but the second problem was what was giving you the error message.

I changed the following:
* There is now also an error message for non-existing glosses
* You now see all errors at once, instead of only the last one
